.circleci/config.yml

```diff
@@ -28,35 +39,38 @@ jobs:
           when: on_success
           paths:
             - ~/.local/share/virtualenvs/
-          key: v2-dependencies-{{ checksum "Pipfile.lock" }}-{{ checksum "thumbs/Pipfile.lock" }}
-
+          key: v3-dependencies-{{ checksum "Pipfile.lock" }}-{{ checksum "thumbs/Pipfile.lock" }}
   run-tests:
     docker:
-      - image: circleci/python:3.8.6
+      - image: cimg/python:3.8
+        environment:
+          CIRCLECI: true
+          PGHOST: 127.0.0.1
+      - image: cimg/postgres:16.3
+        environment:
+          POSTGRES_USER: electionleaflets
+          POSTGRES_DB: electionleaflets
     working_directory: ~/repo
+
     steps:
       - checkout
+
+      # Download and cache dependencies
       - restore_cache:
           keys:
-            - v2-dependencies-{{ checksum "Pipfile.lock" }}
-            - v2-dependencies-default
-      - run: *install-pipenv
-#      - run:
-#          command: pipenv run black-check
-#          name: Black
```

Comment:
Are we binning black/flake here because we're planning to just move to ruff in the next PR?

Reply:
Yes, exactly.